src/client.rs

```diff
@@ -375,7 +376,7 @@ impl Client {
                 let text = auth_res.text().await?;
                 debug!("Received response from auth request: {}", text);
                 let token: RegistryToken = serde_json::from_str(&text)
-                    .context("Failed to decode registry token from auth request")?;
+                    .map_err(|e| OciDistributionError::RegistryTokenDecodeError(e.to_string()))?;
                 debug!("Succesfully authorized for image '{:?}'", image);
```

Comment:
typo (not introduced by this PR, but noticed anyway): _Succesfully_

Reply:
fixed, good catch